### DISCUSSION TITLE
libgit2 summit

### DISCUSSION THREAD

**Discussion by ethomson** [libgit2/libgit2#6674] [General]
@csware suggested that we chat about libgit2. I would be very happy to chat with _any_ user of libgit2 about how they're using the library so that I can understand what they're doing and how I can better assist.

But I'd also be very interested to see if we should have some sort of get together - a libgit2 summit of sorts. It's been several years since we did this (last timed with Git Merge). I can talk to a roadmap for libgit2, and we can have some free form sessions as well.

Almost certainly this will need to largely be online / virtual. But I'm curious if anybody would want to get together in person? Maybe in London?

**Comment by Uncommon** [comment]
I wouldn't be able to attend in person unless it were local (Salt Lake City area, which I'm sure is a longshot 😁) but I'd definitely be interested in participating onilne.

**Comment by csware** [comment]
At least for the first meeting, I'd vote for an online meeting.
